src/apps/js_generic/js_generic.cpp

```diff
@@ -507,9 +521,10 @@ namespace ccfapp
         }
 
         // Call exported function
-        int argc = 0;
-        JSValueConst* argv = nullptr;
+        int argc = 1;
+        JSValueConst* argv = (JSValueConst *)&request;
```

Comment:
Minor: this is supposed to be a (C-style) array, can we make it look more like one? Something like `JSValueConst argv[] = {&request};`?

Reply:
I don't think it works like that, compiler is complaining as well: `a value of type "JSValue *" cannot be used to initialize an entity of type "int32_t"`.